net-libs/libbitcoinconsensus/libbitcoinconcensus-0.15.1.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="Original Bitcoin client"
+DESCRIPTION="BitCoin Core is the backbone of the network. It downloads and, \
+by default, stores the entire history of Bitcoin transactions (which is \
+currently more than 100 GBs); depending on the speed of your computer and \
+network connection, the synchronization process can take anywhere from a few \
+hours to a day or more."
+HOMEPAGE="https://bitcoincore.org"
```

Comment:
```diff
-HOMEPAGE="https://bitcoincore.org"
+HOMEPAGE="https://bitcoincore.org/"
```

BTW, since we all seem to hit the same error:
```
configure: error: No working boost sleep implementation found.
```
if you would like to give this PR a rest until we get some help, that's OK.
Just mark all architectures as broken and we might commit this as is in a couple of days if we did not find anything interesting.